americangut/make_phyla_plots.py

```diff
@@ -14,6 +13,9 @@
 from operator import itemgetter
 import colorbrewer
 
+
+use('agg')
```

Comment:
You could leave this where it was (assuming moving this line may cause trouble later) by adding `#  noqa` at the end of that line. This tells flake8 to ignore any errors raised by this line.